projects.yaml

```diff
@@ -554,6 +554,13 @@ projects:
   category: datasets
   description: Database of yet-to-be-sythesized materials predicted using state-of-the-art machine learning algorithms
   homepage: https://matterverse.ai/
+- name: Open Databases Integration for Materials Design (OPTIMADE)
+  resource: true
+  category: datasets
+  description: Specification of a common REST API for access to 25+ materials databases
+  homepage: https://optimade.org
+  license: https://github.com/Materials-Consortia/OPTIMADE?tab=CC-BY-4.0-1-ov-file#readme
```

Comment:
Will replace `license: ...` with `license: CC-BY-4.0`. The CI/CD knows how to process standard licenses.